app/renderer/components/common/messageBox.js

```diff
@@ -162,34 +161,31 @@ class MessageBox extends React.Component {
 }
 
 const styles = StyleSheet.create({
-  dialog: {
-    outline: 'none'
```

Comment:
Dialog has this style thanks to https://github.com/brave/browser-laptop/commit/2c3126f6f641ab9762f36dad558580e379667cf9#diff-5d3d90fdb0c3efa6acd09057f2d4b101R85 (messageBox is focused by default).